akka-remote/src/test/scala/akka/remote/artery/ArteryMultiNodeSpec.scala

```diff
@@ -41,11 +41,7 @@ abstract class ArteryMultiNodeSpec(config: Config)
 
   override protected def withFixture(test: NoArgTest): Outcome = {
     // note that withFixture is also used in FlightRecorderSpecIntegration
-    if (!RARP(system).provider.remoteSettings.Artery.Enabled) {
-      info(s"${getClass.getName} is only enabled for Artery")
-      Pending
-    } else
-      super.withFixture(test)
+    super.withFixture(test)
```

Comment:
Do we still need the fixture if it doesn't do anything except call super?

Reply:
removed f232b9f10f